package/kedro_viz/utils/api_tools.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+import fsspec
+from fastapi.encoders import jsonable_encoder
+from kedro.io.core import get_protocol_and_path
+
+from kedro_viz.api.rest.responses import (
+    EnhancedORJSONResponse,
+    get_default_response,
+    get_node_metadata_response,
+    get_selected_pipeline_response,
+)
+from kedro_viz.data_access import data_access_manager
+
+logger = logging.getLogger(__name__)
+
+
+def save_api_responses_to_fs(filepath: str):
```

Comment:
can we have save_api_responses_to_fs inside -> `package/kedro_viz/api/rest/responses.py ` as this is also creating some kind of response ?